Dockerfile

```diff
@@ -6,12 +6,13 @@ COPY src ./src
 RUN mvn versions:set -DnewVersion=${version} && \
   mvn package -P assembly
 
-FROM adoptopenjdk/openjdk13:alpine-jre
+FROM adoptopenjdk/openjdk14:alpine-jre
+ENV JAVA_OPTS=""
 LABEL description="Artipie binary repository managment tool"
 LABEL maintainer="titantins@gmail.com"
 LABEL maintainer="g4s8.public@gmail.com"
 COPY --from=build_jar /jar/target/artipie-jar-with-dependencies.jar /usr/lib/artipie.jar
 COPY _config.yml /etc/artipie.yml
 EXPOSE 80
-ENTRYPOINT ["java", "-jar", "/usr/lib/artipie.jar"]
+ENTRYPOINT java $JVM_OPTS --enable-preview -XX:+ShowCodeDetailsInExceptionMessages -jar /usr/lib/artipie.jar
```

Comment:
@Sammers21 you've introduced env variable called `JAVA_OPTS` but uses `JVM_OPTS`